src/components/Scheme/SchemeContainer.tsx

```diff
@@ -96,7 +101,13 @@ class SchemeContainer extends React.Component<IProps, null> {
 
           <div className={css.schemeMenu}>
             <Link className={proposalsTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/proposals/`}>Proposals</Link>
+
+            { // if Bounties Scheme, create new tab
+              (schemeName(schemeState, schemeState.address) === "Standard Bounties") &&
+              <Link className={openTabClass} to={`/dao/${daoAvatarAddress}/scheme/${schemeId}/open/`}>Open Bounties</Link>
```

Comment:
I think this url should be less generic and contain something specific to this contract, like `/dao/${daoAvatarAddress}/scheme/${schemeId}/openbounties/`

Reply:
Updated in 7c89ec3